src/blurredLocation.js

```diff
@@ -59,6 +59,22 @@ BlurredLocation = function BlurredLocation(options) {
 
   // options.map.setView([options.location.lat, options.location.lon], options.zoom);
   options.scaleDisplay = options.scaleDisplay || "scale";
+  options.blurryScale = options.blurryScale || "scale_blurry"
+
+  options.blurryScaleDict = options.blurryScaleDict || {
+   "urban_0":"country",
```

Comment:
What if we had two sub-objects, so you could call `blurreScaleNames.urban[1]` or `.rural[5]`?

Reply:
Done!